openbb_terminal/etf/fmp_model.py

```diff
@@ -0,0 +1,48 @@
+"""FinancialModelingPrep model"""
+__docformat__ = "numpy"
+
+import json
+import logging
+from typing import Dict
+from urllib.error import HTTPError
+from urllib.request import urlopen
+
+from openbb_terminal import config_terminal as cfg
+from openbb_terminal.decorators import log_start_end
+from openbb_terminal.rich_config import console
+
+logger = logging.getLogger(__name__)
+
+# pylint: disable=consider-using-with
+
+
+@log_start_end(log=logger)
+def get_etf_sector_weightings(name: str) -> Dict:
+    """Return sector weightings allocation of ETF. [Source: FinancialModelingPrep]
+
+    Parameters
+    ----------
+    name: str
+        ETF name
+
+    Returns
+    -------
+    Dict[str, Any]
+        Dictionary with sector weightings allocation
+    """
+    try:
+        response = urlopen(
```

Comment:
really had to be difficult and go with urllib instead of requests lol

Reply:
Oh lol actually just copy pasted from FundamentalAnalysis package

Reply:
is this a free endpoint or nah ?

Reply:
Yeah it's free but for some tickers it aint.

Reply:
```
(🦋) /etf/ $ load VOO

Loading Daily data for VOO with starting period 2022-02-08. 


(🦋) /etf/ $ weights


(🦋) /etf/ $ weights --raw

                                   
      Sector holdings of VOO       
┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃ Sector                 ┃ Weight ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ Real Estate            │ 2.47%  │
├────────────────────────┼────────┤
│ Consumer Cyclical      │ 12.21% │
├────────────────────────┼────────┤
│ Basic Materials        │ 2.34%  │
├────────────────────────┼────────┤
│ Consumer Defensive     │ 6.59%  │
├────────────────────────┼────────┤
│ Technology             │ 23.5%  │
├────────────────────────┼────────┤
│ Communication Services │ 10.93% │
├────────────────────────┼────────┤
│ Financial Services     │ 14.18% │
├────────────────────────┼────────┤
│ Utilities              │ 2.65%  │
├────────────────────────┼────────┤
│ Industrials            │ 9.22%  │
├────────────────────────┼────────┤
│ Energy                 │ 2.79%  │
├────────────────────────┼────────┤
│ Healthcare             │ 13.13% │
└────────────────────────┴────────┘

(🦋) /etf/ $ load SPY

Loading Daily data for SPY with starting period 2022-02-08. 


(🦋) /etf/ $ weights --raw

                                   
      Sector holdings of SPY       
┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃ Sector                 ┃ Weight ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ Real Estate            │ 2.53%  │
├────────────────────────┼────────┤
│ Consumer Cyclical      │ 12.43% │
├────────────────────────┼────────┤
│ Basic Materials        │ 2.34%  │
├────────────────────────┼────────┤
│ Consumer Defensive     │ 6.4%   │
├────────────────────────┼────────┤
│ Technology             │ 23.44% │
├────────────────────────┼────────┤
│ Communication Services │ 11.18% │
├────────────────────────┼────────┤
│ Financial Services     │ 14.41% │
├────────────────────────┼────────┤
│ Utilities              │ 2.64%  │
├────────────────────────┼────────┤
│ Industrials            │ 9.05%  │
├────────────────────────┼────────┤
│ Energy                 │ 2.68%  │
├────────────────────────┼────────┤
│ Healthcare             │ 12.89% │
└────────────────────────┴────────┘

(🦋) /etf/ $ load VTI

wLoading Daily data for VTI with starting period 2022-02-08. 


(🦋) /etf/ $ weights --raw

                                   
      Sector holdings of VTI       
┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃ Sector                 ┃ Weight ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ Real Estate            │ 3.51%  │
├────────────────────────┼────────┤
│ Consumer Cyclical      │ 12.08% │
├────────────────────────┼────────┤
│ Basic Materials        │ 2.54%  │
├────────────────────────┼────────┤
│ Consumer Defensive     │ 6.05%  │
├────────────────────────┼────────┤
│ Technology             │ 22.9%  │
├────────────────────────┼────────┤
│ Communication Services │ 10.29% │
├────────────────────────┼────────┤
│ Financial Services     │ 13.98% │
├────────────────────────┼────────┤
│ Utilities              │ 2.51%  │
├────────────────────────┼────────┤
│ Industrials            │ 9.78%  │
├────────────────────────┼────────┤
│ Energy                 │ 2.64%  │
├────────────────────────┼────────┤
│ Healthcare             │ 13.73% │
└────────────────────────┴────────┘

(🦋) /etf/ $
```

Reply:
nice.  was curious